src/main/resources/resources/logisim/strings/std/std_cn.properties

```diff
@@ -858,6 +858,7 @@ pinPullAttr = 拉动行为
 pinPullDownOption = 下拉
 pinPullNoneOption = 不变
 pinPullUpOption = 向上拉起
+# pinResetValue = 
```

Comment:
> `# pinResetValue =`

Don't we rather use `==>` prefixed keys (multiple places)?

    # ==> pinResetValue =

Reply:
Done

Reply:
Sorry, but there is now a small problem that my pushes to my fork are now related to another repro. Have to sort that out.

Reply:
Now it is fixed